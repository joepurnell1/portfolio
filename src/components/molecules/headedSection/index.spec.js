import React from 'react';
import { ThemeProvider } from 'styled-components';
import renderer from 'react-test-renderer';
import HeadedSection from './index';

const theme = {
  Global: {
    backgroundColour: '#737215',
  },
  HeadedSection: {
    headerText: '#100',
  },
};

describe('headed section component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <ThemeProvider theme={theme}>
        <HeadedSection
          sectionTitle="Rhymes like ours could never be stopped"
        >
          <HeadedSection>Party rockers, non-stoppers, and our names are def</HeadedSection>
        </HeadedSection>
      </ThemeProvider>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
