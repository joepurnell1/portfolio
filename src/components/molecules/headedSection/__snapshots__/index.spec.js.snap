// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`headed section component should render default as expected 1`] = `
<div
  className="sc-bxivhb jBrEJL sc-bwzfXH uTxCW sc-bdVaJa cTPAwo"
>
  <h1
    className="sc-htpNat Wqnyr"
  >
    Rhymes like ours could never be stopped
  </h1>
  <div
    className="sc-bxivhb jBrEJL sc-bwzfXH uTxCW sc-bdVaJa cTPAwo"
  >
    <h1
      className="sc-htpNat Wqnyr"
    >
      
    </h1>
    Party rockers, non-stoppers, and our names are def
  </div>
</div>
`;
